xml/dolly.xml

```diff
@@ -0,0 +1,362 @@
+  <title>Dolly configuration file</title>
+  <para>
+   <command>dolly</command> can also use a configuration file for the cloning process.
+   The format is strict but simple. Following is a typical configuration file example:
+  </para>
+  <screen>
+infile /tmp/sle15sp3.sif<co xml:id="dolly-config-in"/>
+outfile /data/sle15sp3.sif<co xml:id="dolly-config-out"/>
+server sle15sp31<co xml:id="dolly-config-server"/>
+firstclient sle15sp32<co xml:id="dolly-config-first"/>
+lastclient sle15sp34<co xml:id="dolly-config-last"/>
+clients 3<co xml:id="dolly-config-nbclients"/>
+sle15sp32<co xml:id="dolly-config-nodes"/>
+sle15sp33
+sle15sp34
+endconfig<co xml:id="dolly-config-end"/></screen>
+  <calloutlist>
+   <callout arearefs="dolly-config-in">
+    <para>
+     <literal>infile <replaceable>DATA</replaceable></literal> the data you send over the network
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-out">
+    <para>
+     <literal>outfile <replaceable>DATA</replaceable></literal>: file or device to store data on the client node
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-server">
+    <para>
+     <literal>server <replaceable>MASTER_NODE</replaceable></literal>: name of the master node
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-first">
+    <para>
+     <literal>firstclient <replaceable>NODE</replaceable></literal>: first client on the ring
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-last">
+    <para>
+     <literal>lastclient <replaceable>NODE</replaceable></literal>: last client in the ring
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-nbclients">
+    <para>
+     <literal>clients <replaceable>NUMBER</replaceable></literal> how many dolly clients are in the ring
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-nodes">
+    <para>
+     <literal><replaceable>NODENAME</replaceable></literal>: the list of dolly client, one per line
+    </para>
+   </callout>
+   <callout arearefs="dolly-config-end">
+    <para>
+     <literal>endconfig</literal>: mandatory parameter to specify the end of the configuration file
+    </para>
+   </callout>
+  </calloutlist>
+  <para>The only change is on the master node, you need to use the <option>-f <replaceable>CONFIGURATION_FILE</replaceable></option> on the <command>dolly</command> command line:</para>
+  <screen>&prompt; dolly -v -s<co xml:id="dolly-config-command"/> -f /etc/dolly.cfg</screen>
+  <calloutlist>
+   <callout arearefs="dolly-config-command">
+    <para><option>-s</option> is mandatory if you are using <option>-f</option></para>
+   </callout>
+  </calloutlist>
+ </sect1>
+
+ <sect1 xml:id="dolly-support">
+ <title>Dolly limitations</title>
+ <para>
+  <command>dolly</command> provides some specifics parameters which are not supported because
+  they are unsafe, untested, dedicated for benchmarking or doing statistics. They are provided
+  as technology preview.
```

Comment:
```suggestion
  <command>dolly</command> provides some specific parameters which are not supported because
  they are unsafe, untested, or dedicated for benchmarking or doing statistics. They are provided
  as a technology preview only.
```